reference/comparison-table.css

```diff
@@ -8,10 +8,14 @@ Would be good to fix this, or to make the page wider.
 
 */
 .comparison {
-    table-layout: fixed;
+    display: flex;
+    justify-content: center;
```

Comment:
Do these make any practical difference? (I have no strong bond to the existing JS; I'm curious because I'm trying to learn how to make these better)

Reply:
I added a container div, so the editor can be wider than the content column.

This is css for centering the table, even when it overflows.